ext/node/resolution.rs

```diff
@@ -187,13 +186,8 @@ impl NodeResolver {
       }
     }
 
-    let url = self.module_resolve(
-      specifier,
-      referrer,
-      DEFAULT_CONDITIONS,
-      mode,
-      permissions,
```

Comment:
Are you sure these were only used for package.json? I thought we used them when probing for various file extensions

Reply:
It hasn't been from what I can tell, but even if it were I don't think we should bother permission checking that?